src/payment/payment-method-mappers/payment-method-id-mapper.js

```diff
@@ -46,6 +56,10 @@ export default class PaymentMethodIdMapper {
             return BRAINTREE;
         }
 
+        if (isPaypalCommercePaymentMethod(id)) {
```

Comment:
for clarity, should this method be called `isPaypalCommerceCreditPaymentMethod` ?

Reply:
In this method PayPal Commerce is checked. This method can be extended in the future and check not only credit (like braintree)
So I think `isPaypalCommerceCreditPaymentMethod` is not so correct